### ISSUE TITLE
9.1.3.1a/8.2 The page has two identical headings labeled "Availability" which have been marked as "h1" and "h2" headings. This should be avoided.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42273]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-be75bbae42-v8-n2

9.1.3.1a/8.2 The page has two identical headings labeled "Availability" which have been marked as "h1" and "h2" headings. This should be avoided.

**Comment by JuliaKirschenheuter** [comment]
duplicate of https://github.com/nextcloud/server/issues/41826
